ledger-core/conveyor/conveyor_semaphore.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package conveyor
+
+import (
+	"runtime"
+
+	"github.com/insolar/assured-ledger/ledger-core/conveyor/smachine"
+	"github.com/insolar/assured-ledger/ledger-core/conveyor/smachine/smsync"
+)
+
+type ParallelProcessingLimiter interface {
+	SyncLink() smachine.SyncLink
+	NewChild(childValue int, name string) smsync.SemaChildLink
+}
+
+func NewParallelProcessingLimiter(maxTasks int) ParallelProcessingLimiter {
+	if maxTasks <= 0 {
+		maxTasks = runtime.NumCPU() - 1
+	}
+	if maxTasks < 1 {
+		maxTasks = 1
+	}
+	return &conveyorSemaphore{
+		semaphore: smsync.NewSemaphoreWithFlags(maxTasks, "ConveyorParallelProcessingLimiter", smsync.QueueAllowsPriority),
```

Comment:
1. factually, this is a limiter for runner, not for conveyor
2. make the name an exported constant
3. add a helper method to receives this link pls

Reply:
done